index.html

```diff
@@ -188,7 +195,12 @@ <h2>Scope</h2>
         </ol>
         <div>
           <h3>Success Criteria</h3>
-          <p>In order to advance to <a href="https://www.w3.org/Consortium/Process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, each specification is expected to have <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least two independent implementations</a> of each of feature defined in the specification.</p>
+          <p>In order to advance to 
+            <a href="https://www.w3.org/Consortium/Process/#RecsPR"
+            title="Proposed Recommendation">Proposed Recommendation</a>,
+            each specification is expected to have
```

Comment:
Change "specification" to "specification, i.e., either base document or module document,".

Reply:
I'm going to tackle this a slightly different way, to refer to Recommendation track technical reports instead, which cover all the specification documents we publish without getting into the weeds of whether each is a specification, a module, a base, or whatever.